lib/portage/dbapi/__init__.py

```diff
@@ -38,7 +39,7 @@ def __init__(self):
         pass
 
     @property
-    def categories(self):
+    def categories(self) -> Sequence[str]:
```

Comment:
This always returns a tuple so why not just say so? :)

The annotations you are using instead, allow `return 'a simple string'` because the str type is a sequence of str (if you try to act on a str as a sequence, it acts like `['a', ' ', 's', 'i', 'm', 'p', 'l', 'e', ' ', 's', 't', 'r', 'i', 'n', 'g']`).

Reply:
I kind of forgot that that can be the case, specifically for `Sequence[str]`. Fixing this.